media_driver/agnostic/gen9_kbl/codec/kernel/media_srcs.cmake

```diff
@@ -18,9 +18,13 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 
-set(TMP_SOURCES_
-    ${CMAKE_CURRENT_LIST_DIR}/igcodeckrn_g9_kbl.c
-)
+if(${FULL_OPEN_BUILD} STREQUAL "no")
+
+    set(TMP_SOURCES_
+        ${CMAKE_CURRENT_LIST_DIR}/igcodeckrn_g9_kbl.c
```

Comment:
What about HUC?

Reply:
hi @dvrogozh , why we need to care about HUC? we have some huc relevant code, but it is just valid when there are Huc firmware. it just like: there are no MBEnc media kernel with full_open_source, but the relevant code still there, just does not take effect.

Reply:
For Q1: What will be the behavior when user will try to use a feature which requires the shader? Please, describe this in a commit message.
Answer: Yes, we could add these options support in the next steps discussion.

Reply:
@XinfengZhang : we need care about HUC shaders because this invalidates "free" media driver - you have a binary blob inside the driver and that's actually it. This will:
1. Raise questions whether you actually use it at runtime or not. We will _need_ to explain. This at least gives delays in the discussions.
2, Raise suspicious whether we accidentally may try to cheat the Linux kernel requirements to have open source user.

IMHO, it is just simpler to strip it our _explicitly_. Let's avoid being asked questions when we can avoid them altogether.

Reply:
@yangnin : you did not answer on the question and it is critical one actually. I repeat: "What will be the behavior when user will try to use a feature which requires the shader? Please, describe this in a commit message."